src/styles/global.scss

```diff
@@ -46,3 +46,7 @@ a:active {
     font-family: $stockkeeper-primary-font;
   }
 }
+
```

Comment:
also, can we sort these properties alphabetically in global.scss; easier to find later .. hopefully